core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -392,7 +392,7 @@ object ConsoleConsumer extends Logging {
   private[tools] class ConsumerWrapper(topic: Option[String], partitionId: Option[Int], offset: Option[Long], whitelist: Option[String],
                                        consumer: Consumer[Array[Byte], Array[Byte]], val timeoutMs: Long = Long.MaxValue) {
     consumerInit()
-    var recordIter = Collections.emptyList[ConsumerRecord[Array[Byte], Array[Byte]]]().iterator()
+    var recordIter = consumer.poll(Duration.ofMillis(0L, false)).iterator
```

Comment:
Maybe `consumer.poll(Duration.ofMillis(0L, false))` -> `consumer.poll(Duration.ofMillis(0L), false)`?

Reply:
@HeartSaVioR OMG, it's a typo. Thanks for letting me know! :)